index.html

```diff
@@ -26,162 +26,189 @@
+        <button type="button" class="btn1">Get My Resume</button>
       </div>
-        <ul>
-          <li>Codekit</li>
-          <li>Github</li>
-          <li>Codepen</li>
-          <li>Gitlab</li>
-          <li>Terminal</li>
-        </ul>
-    </section> 
-  </section>
-  <section class="contact-form-1">
-    <div class="contact-header">
-      <h3 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-    </div>
-    <form action="https://formspree.io/f/xjvjrbpg" method="POST">
-      <div class="contact-inputs">
-        <input type="text" id="name" name="name" maxlength="30" placeholder="Full Name" required>
+      <hr>
+      <div class="knowledge">
+        <div class="languages">
+          <div class="knowledge-titles">
+            <img src="img/Rectangle-55.png" alt="">
+            <h3>Language</h3>
+          </div>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="frameworks">
+          <div class="knowledge-titles">
+            <img src="img/Rectangle-56.png" alt="">
+            <h3>Frameworks</h3>
+          </div>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <div class="knowledge-titles">
+            <img src="img/Rectangle-57.png" alt="">
+            <h3>Skills</h3>
+          </div>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+       </div>
+      </div> 
+    </section>
+    <section class="contact-form-1">
+      <div class="contact-header">
+        <h3 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
       </div>
-      <div class="contact-inputs">
-        <input type="email" id="email" name="email" placeholder="Email address" required>
+      <form action="https://formspree.io/f/xjvjrbpg" method="POST">
+        <div class="contact-inputs">
+          <input type="text" id="name" name="name" maxlength="30" placeholder="Full Name" required>
+        </div>
+        <div class="contact-inputs">
+          <input type="email" id="email" name="email" placeholder="Email address" required>
+        </div>
+      <textarea class="contact-message" id="message" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+      <div class="form-button">
+        <button type="submit" id="submit" class="btn1">Get in touch</button>
```

Comment:
- [ ] The bold text `I'm always interested in hearing about new projects, ....` should share the same row with the form but yours isn't well aligned. Kindly fix it.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/121673758-27be5180-caa9-11eb-90d4-03c1f8afc7c2.png) | ![image](https://user-images.githubusercontent.com/16058470/121673950-60f6c180-caa9-11eb-93d5-2f670c6d31e6.png)